src/main/java/picard/TestStringUtilSplit.java

```diff
@@ -6,20 +6,21 @@
 import java.util.StringTokenizer;
 
 /**
- *
+ * A class that can be used to check which is faster, StringUtil.split or StringTokenizer.
+ * Currently, StringUtil is faster.
  */
-public class Test {
+public class TestStringUtilSplit {
```

Comment:
If this is a Test then it should be in the test source tree.

Reply:
Leaving it in enables us to run in from within the main jar. given that it isn't being run during the tests, I think that's a better option.

Reply:
Don't we have a test .jar? If not, we should be able to generate one as part of the build.

If this is only run as part of a test, why does it need to be included in the main .jar? I'm not sure why this is being committed to the picard repo in the first place; it looks like test code that was run once to show that X was faster than Y. Maybe the test here is an assert that this speed difference continues to be true?

In general, do we have a way to annotate classes that are public and part of the main .jar but are not intended to be used (e.g. not part of the picard API)?